tekster/nb/aktivitetkort.dato_label_nb.txt

```diff
@@ -1,4 +1,5 @@
 { label, select,
 FRA {Fra:}
 TIL {Til:}
-} {DATO}
\ No newline at end of file
+other { {label}:}
```

Comment:
Blir dette brukt? (lurer på alle .txt filene som er committa)

Reply:
Dette kommer fra en annen PR, men ble med pga squash&merge. 
https://github.com/navikt/aktivitetsplan/pull/783.
Ny versjon av intl (formatjs) krever visst fallback ved select. Hvis ikke får du sike feil som https://nav-it.slack.com/archives/GKJ0TTML3/p1620829570054800?thread_ts=1620803457.043400&cid=GKJ0TTML3